<launch>
  <node name="cobot_pick_calibrate" pkg="cobot_pick" type="cobot_pick_calibrate" respawn="false"
         output="screen"  launch-prefix="gdb -ex run --args">
    <param name="load_mode" value="true"/>
    <param name="save_file_prefix" 
      value="/home/tong/catkin_ws/src/cobot/cobot_pick/data/calib"/>
    <param name="show_result" value="true"/>

    <param name="chess_num_x" value="10"/>
    <param name="chess_num_y" value="7"/>

    <param name="chess_size_x" value="0.0232"/>
    <param name="chess_size_y" value="0.0232"/>

    <param name="chess_pos1_x" value="0.0"/>
    <param name="chess_pos1_y" value="0.0"/>
    <param name="chess_pos1_z" value="0.0"/>

    <param name="chess_pos2_x" value="0.0"/>
    <param name="chess_pos2_y" value="0.0"/>
    <param name="chess_pos2_z" value="0.0"/>

    <param name="chess_pos3_x" value="0.0"/>
    <param name="chess_pos3_y" value="0.0"/>
    <param name="chess_pos3_z" value="0.0"/>

    <param name="chess_pos4_x" value="0.0"/>
    <param name="chess_pos4_y" value="0.0"/>
    <param name="chess_pos4_z" value="0.0"/>

    <param name="plane_ransac_repeat_time" value="1000"/>
    <param name="plane_ransac_th_error" value="0.0005"/>
  </node>
  <!--
  <node name="tf_realsense" pkg="tf" type="static_transform_publisher" 
    args="0.349052 0.314813 0.504964 0.560965 0.793735 -0.226977 -0.061516 world my_frame 100" respawn="false" output="screen" />
  -->
</launch>
